tuned/plugins/plugin_audio.py

```diff
@@ -12,10 +12,28 @@
 
 class AudioPlugin(base.Plugin):
 	"""
-	Plugin for tuning audio cards powersaving options.
-
-	Power management is supported per module, not device. From this reason,
-	we take kernel module names as device names.
+	`audio`::
+	
+	Sets audio cards powersaving options. The plug-in sets the autosuspend
+	timeout for audio codecs to the value specified by the [option]`timeout`
```

Comment:
If powersaving and autosuspend are literal option names, use the sign `` around both the terms. If not, there needs to a space between power saving and auto suspend.